src/lib/coerce.js

```diff
@@ -100,6 +100,8 @@ exports.valObjectMeta = {
         requiredOpts: [],
         otherOpts: ['dflt', 'min', 'max', 'arrayOk'],
         coerceFunction: function(v, propOut, dflt, opts) {
+            if(isTypedArraySpec(v)) v = decodeTypedArraySpec(v);
```

Comment:
Fix for handling `base64` encoded arrays. 
cc: #5230 and https://github.com/plotly/plotly.py/pull/4470